kong/dao/migrations/postgres.lua

```diff
@@ -692,4 +694,76 @@ return {
     ]],
     down = nil
   },
+  {
+    name = "2018-03-27-123400_prepare_certs_and_server_names",
+    up = [[
+      DO $$
+      BEGIN
+        ALTER TABLE ssl_certificates    RENAME TO certificates;
+        ALTER TABLE ssl_servers_names   RENAME TO server_names;
+      EXCEPTION WHEN duplicate_table THEN
+        -- Do nothing, accept existing state
+      END$$;
+
+      DO $$
+      BEGIN
+        ALTER TABLE server_names RENAME COLUMN ssl_certificate_id TO certificate_id;
+        ALTER TABLE server_names ADD    COLUMN id uuid;
+      EXCEPTION WHEN undefined_column THEN
+        -- Do nothing, accept existing state
+      END$$;
+    ]],
+    down = nil
+  },
+  {
+    name = "2018-03-27-125400_fill_in_server_names_ids",
+    up = function(_, _, dao)
+      local rows, err = dao.db:query([[
+        SELECT * FROM server_names;
+      ]])
+      if err then
+        return err
+      end
+
+      local fmt = string.format
+
+      for _, row in ipairs(rows) do
+        local sql = fmt("UPDATE server_names SET id = '%s' WHERE name = '%s';",
+                        utils.uuid(),
+                        row.name)
+        local _, err = dao.db:query(sql)
```

Comment:
Instead of executing these one-by-one, could up make up an transaction and do them in one `query` that is wrapped in transaction?